drivers/wifi/eswifi/eswifi_core.c

```diff
@@ -119,6 +119,50 @@ static void __parse_scan_res(char *str, struct wifi_scan_result *res)
 	}
 }
 
+int eswifi_at_cmd_rsp(struct eswifi_dev *eswifi, char *cmd, char **rsp)
```

Comment:
Should `cmd` be `const char *` here?

Reply:
cmd string can be altered by the spi layer (byte stuffing).